server/common.js

```diff
@@ -141,7 +141,33 @@ var isEmptyObject = function(obj) {
 exports.isEmptyObject = isEmptyObject;
 
 // return the current date
-function getDate() {
-    return date().format('YYYY-MM-DD hh:mm:ss A');
+var getDate = function() {
+    return getDateByFormat('YYYY-MM-DD hh:mm:ss A');
 }
+
+// return the current date with format
+var getDateByFormat = function(format) {
+    return date().format(format);
+}
+
 exports.getDate = getDate;
+exports.getDateByFormat = getDateByFormat;
+
+// This is the function.
```

Comment:
better comment?

Reply:
what the hell? did I actually right this?! will fix